### PR TITLE
Marketplace pictures slideshow

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/39373

## Description
Add three banners in the marketplace page in xsoar.pan.dev,
under the filter bar as a slideshow, each link to other site.

## Screenshots
image

### CODE REVIEW

**Comment by daryakoval** [comment]
TODO: check if we need remove scroller near the pictures.

**Review by dantavori — APPROVED** [review]

**Review by dantavori — COMMENTED** [review]
please remove the .demisto-sdk.report file

File: .demisto-sdk-report [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (glicht) [inline .demisto-sdk-report:1]: Please remove this file

File: src/pages/marketplace/slideshow.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Slide } from 'react-slideshow-image';
+import 'react-slideshow-image/dist/styles.css'
+import Link from "@docusaurus/Link";
+
+
+const slideImages = [
+  {
+    url: 'https://raw.githubusercontent.com/demisto/content-docs/148c5a9e7153b1688acb2f331b414e30a3f75f8c/src/pages/marketplace/images/Marketplace_Vertical_1.png',
```
Comment (glicht) [inline src/pages/marketplace/slideshow.js:9]: Link to the images with relative links and not via the `raw.github..` link. The images should go into the `static/img` dir. You can open there a subdir and put all related images to the sliideshow.

**Review by glicht — CHANGES_REQUESTED** [review]

**Review by glicht — APPROVED** [review]
Looks good

**Pull request merged**